packages/web3-providers/src/NextID/proof.ts

```diff
@@ -112,13 +112,13 @@ export class NextIDProofAPI implements NextIDBaseAPI.Proof {
     }
     async queryAllExistedBindingsByPlatform(platform: NextIDPlatform, identity: string) {
         const nextIDPersonaBindings: NextIDPersonaBindings[] = []
-        let page = 0
+        let page = 1
```

Comment:
Does this mean the service changed the pagination index?

Reply:
https://github.com/nextdotid/proof-server/blob/0065ed83f8cb9def1750718272aac5472918f0a8/docs/api.apib#L166

Reply:
CC @lelenei So this bug existed before.